README.md

```diff
@@ -23,21 +23,38 @@
 # Apache Daffodil™ Extension for Visual Studio Code
 
 [![CI](https://github.com/apache/daffodil-vscode/workflows/CI/badge.svg)](https://github.com/apache/daffodil-vscode/actions/workflows/CI.yml)
+[![Nightly Tests](https://github.com/apache/daffodil-vscode/actions/workflows/nightly.yml/badge.svg)](https://github.com/apache/daffodil-vscode/actions/workflows/nightly.yml)
 
 </div>
 
 This is an extension for Visual Studio Code (VS Code) which enables the interactive debugging of DFDL schema parsing using [Apache Daffodil](https://daffodil.apache.org/).
 
 ## Build Requirements
 
-- Java Development Kit (JDK) 11 or higher
+- Java Development Kit (JDK) 8 or higher
 - SBT 0.13.8 or higher
 - Node 10 or higher
 - Yarn (https://yarnpkg.com/getting-started/install)
 
-## Download - Prebuilt
+## Download
 
-Until the extension is available in the [VS Code Extension Marketplace](https://marketplace.visualstudio.com/vscode), please download the latest `.vsix` file from the [releases page](https://github.com/apache/daffodil-vscode/releases).
+### VS Code Marketplace
+
+The easiest way to install the extension is through the VS Code Extension Marketplace. The steps are as follow:
+
+- Open VS Code.
+- Navigate to extension marketplace (icon looks like stacked blocks).
+- Search for `Apache Daffodil VS Code Extension`.
+- The first result will be the item you want, click `Install`.
```

Comment:
Thoughts on just including the instructions that markplace uses? e.g. `Ctrl+P` and run `ext install ASF.apache-daffodil-vscode`. Avoids having to describe icons that might change, or potenial ambiguities if the search ever finds some other extension or for some reason ours isn't first.

Reply:
Yep :+1: